README.md

```diff
@@ -138,8 +138,9 @@ This module exports a [plugin registration function](https://github.com/fastify/
 -   `bufferFlushTimeout`: Number. Metrics buffer flush timeout. See [dats](https://github.com/immobiliare/dats#new-clientoptions).
 -   `sampleInterval`: Number. Optional. Sample interval in `ms` used to gather process stats. Defaults to `1000`.
 -   `onError`: Function: `(err) => void`. Optional. This function to handle possible Dats errors. See [dats](https://github.com/immobiliare/dats#new-clientoptions). Default: `(err) => log(err)`
--   `udpDnsCache`: Boolean. Optional. Activate udpDnsCache. Default `true`.
--   `udpDnsCacheTTL`: Number. Optional. DNS cache Time to live of an entry in seconds. Default `120`.
+-   `udpDnsCache`: Boolean. Optional. Activate udpDnsCache. See [dats](https://github.com/immobiliare/dats#new-clientoptions). Default `true`.
+-   `udpDnsCacheTTL`: Number. Optional. See [dats](https://github.com/immobiliare/dats#new-clientoptions). DNS cache Time to live of an entry in seconds. Default `120`.
+-   `customDatsClient`: Dats instance. Optional. The custom statsd client, It must have the same interface of [dats](https://github.com/immobiliare/dats#new-clientoptions). If set, all dats parameters will not work.
```

Comment:
Are you referring to the dns parameters and similar? If so, I think it is implied and I would omit it. The point of the feature is to allow a custom dats instance, and by dats I mean our client, not a custom StatsD client. At least for  now 😉 . I would rephrase this as:

"A custom dats instance" and point to the documentation of dats as you have done already.

Reply:
Ok, I fix it.